src/packages/@ncigdc/components/AddToCartButtonAll.js

```diff
@@ -89,14 +89,15 @@ const AddToCartButtonAll = ({
         <span style={{ marginLeft: '0.5rem' }}>
           <ShoppingCartIcon
             className="test-toggle-all-files-in-cart"
-            active={inCart}
             onClick={() =>
               inCart
                 ? dispatch(toggleFilesInCart(edges))
                 : dispatch(addAllFilesInCart(edges))}
             aria-label="Add files to cart"
             role="button"
-            style={iconButton}
+            style={
+              (iconButton, { color: inCart ? '#3c763d' : 'rgb(0, 80, 131)' })
```

Comment:
just delete iconButton, unless you meant to spread it into style? But seems like you don't need it.
![image](https://user-images.githubusercontent.com/1314446/38820600-50d6fdac-416c-11e8-9295-1710a1ec2483.png)